src/malloc_wrapper.cc

```diff
@@ -99,7 +99,7 @@ void *__wrap_malloc(size_t size)
 {
 	void *lptr = __real_malloc(size);
 
-	spdlog::info("Malloc: {} bytes @{:p}\n", size, lptr);
```

Comment:
[cppcoreguidelines-pro-bounds-array-to-pointer-decay]: do not implicitly decay an array into a pointer; consider using gsl::array_view or an explicit cast instead